pkg/service/service.go

```diff
@@ -176,36 +178,83 @@ func (s *Service) GetCurrentTs() time.Time {
 	return time.Now()
 }
 
+func (s *Service) populateV2BeMapFromV1BeMap(ipv4, ipv6 bool) error {
```

Comment:
Nit: in the past we tried to get rid of FE and BE terminology, so maybe populateBackendMapV2FromV1?

Reply:
Done.